packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import React, { ElementType } from 'react'
+
+import {
+  SvgJoyTokenMonochrome16,
+  SvgJoyTokenMonochrome24,
+  SvgJoyTokenMonochrome32,
+  SvgJoyTokenMonochrome48,
+  SvgJoyTokenPrimary16,
+  SvgJoyTokenPrimary24,
+  SvgJoyTokenPrimary32,
+  SvgJoyTokenPrimary48,
+  SvgJoyTokenSilver16,
+  SvgJoyTokenSilver24,
+  SvgJoyTokenSilver32,
+  SvgJoyTokenSilver48,
+} from '@/components/_icons'
+
+type JoyTokenIconVariant = 'primary' | 'silver' | 'regular'
+type JoyTokenIconSize = 16 | 24 | 32 | 48
```

Comment:
wouldn't strings make more sense?

Reply:
Is there any advantage to using string?

Reply:
At least for vscode better intellisense/autocomplete

Reply:
Seems to work fine:
<img width="265" alt="Screenshot 2022-03-02 at 17 07 46" src="https://user-images.githubusercontent.com/12646744/156400579-fbe93b3f-8eee-4d75-8aee-f3538a9cee18.png">